### PR TITLE
Added: Use dynamic values for GitHub and Twitter.

### LINKED ISSUE #418: Grab twitter and github url in boilerplate from metadata
Change this https://github.com/MoOx/phenomic/blob/next/boilerplate/web_modules/Header/index.js#L25

Ref https://gitter.im/MoOx/phenomic?at=571e0c07f5b6334a5e70a31c

### PR DESCRIPTION
**Pull request by ben-eb**
Closes #418.

### CODE REVIEW

File: boilerplate/web_modules/app/metadata.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 // you can add any STATIC data you want here
-import pkg from "../../package.json"
+import app from "../../package.json"
+
+const pkg = {
+  ...app,
+  twitter: 'Phenomic_app'
+}
```
Comment (MoOx) [inline boilerplate/web_modules/app/metadata.js]: Not sure about this. Maybe we should put that in the package.json

File: src/bin/commands/setup/questions.js [hunk 2]
```diff
@@ -31,6 +32,18 @@ const questions = [
     },
   },
   {
+    type: "input",
+    name: "twitter",
+    message: "Your project's twitter account",
```
Comment (MoOx) [inline src/bin/commands/setup/questions.js]: maybe we can add "(optional)" here?

**Comment by thangngoc89** [comment]
@ben-eb you broke test here: https://github.com/MoOx/phenomic/blob/505501f0891e76f8d714584b59efc8ae4ad861d9/src/bin/commands/setup/__tests__/inquirer.js

The purpose is for testing inquirer, not testing the questions you can refactor it to use a fixed questions list.

**Comment by thangngoc89** [comment]
Also, you'll need to add test for validating the regex. See example here : https://github.com/MoOx/phenomic/blob/505501f0891e76f8d714584b59efc8ae4ad861d9/src/bin/commands/setup/__tests__/questions.js

**Comment by MoOx** [comment]
@ben-eb do you have some times to update this PR or would you like some help for this?

**Comment by ben-eb** [comment]
Not sure that I'll have time to do this soon, so any help appreciated. 😄

**Comment by thangngoc89** [comment]
I'll merge on local and fix the remaining errors.

**Pull request closed without merging**